diff -urbB spandsp-0.0.2.org/src/echo.c spandsp-0.0.2/src/echo.c
--- spandsp-0.0.2.org/src/echo.c	2005-08-31 21:27:52.000000000 +0200
+++ spandsp-0.0.2/src/echo.c	2005-10-08 23:54:00.942805250 +0200
@@ -89,7 +89,9 @@
 
 #include "spandsp/alaw_ulaw.h"
 #include "spandsp/echo.h"
+#ifdef USE_MMX
 #include "spandsp/mmx.h"
+#endif
 
 #if !defined(NULL)
 #define NULL (void *) 0
diff -urbB spandsp-0.0.2.org/src/spandsp.h spandsp-0.0.2/src/spandsp.h
--- spandsp-0.0.2.org/src/spandsp.h	2005-08-13 12:06:48.000000000 +0200
+++ spandsp-0.0.2/src/spandsp.h	2005-10-08 23:50:04.184008750 +0200
@@ -39,7 +39,9 @@
 #include <spandsp/timing.h>
 #include <spandsp/vector.h>
 #include <spandsp/complex.h>
+#ifdef USE_MMX
 #include <spandsp/mmx.h>
+#endif
 #include <spandsp/arctan2.h>
 #include <spandsp/biquad.h>
 #include <spandsp/fir.h>
